peps/pep-0757.rst

```diff
@@ -4,11 +4,12 @@ Author: Sergey B Kirpichev <skirpichev@gmail.com>,
         Victor Stinner <vstinner@python.org>
 PEP-Delegate: C API Working Group
 Discussions-To: https://discuss.python.org/t/63895
-Status: Draft
+Status: Accepted
 Type: Standards Track
 Created: 13-Sep-2024
 Python-Version: 3.14
 Post-History: `14-Sep-2024 <https://discuss.python.org/t/63895>`__
+Resolution: https://discuss.python.org/t/pep-757-c-api-to-import-export-python-integers/63895/73
```

Comment:
We can now include the date here too:
```suggestion
Resolution: `28-Nov-2024 <https://discuss.python.org/t/pep-757-c-api-to-import-export-python-integers/63895/73>`__
```

Reply:
Oh, interesting, I will do the change when I will mark the PEP as Final.